docs/usage/config.md

```diff
@@ -78,6 +78,7 @@ Please take note of the fact that for a different reference fasta.gz the  unzipp
 | vcf.report.max_records                        |                 |                                                                                                                                                                           |
 | vcf.report.max_samples                        |                 |                                                                                                                                                                           |
 | vcf.report.template                           |                 | for details, see [here](../advanced/report_templates)                                                                                                                     |
+| include_crams                        |true|allowed values: [true, false]. true: include cram files in the report for showing alignments in the genome browser, false: do not include the crams in the report, no aligments are shown in the genome browser. This will result in a smaller report size.|
```

Comment:
- include_crams -->  vcf.report.include_crams
- consistency: move some lines up to maintain alphabetic ordering
- align table orders to stop IntelliJ from complaining